src/Knp/Bundle/KnpBundlesBundle/Entity/Developer.php

```diff
@@ -253,6 +264,37 @@ public function getContributionBundles($page = null, $limit = 15)
     }
 
     /**
+     * Add collaborated Bundle
+     *
+     * @param Bundle $collaboratedBundle
+     */
+    public function addCollaboratedBundle(Bundle $collaboratedBundle)
+    {
+        $this->collaboratedBundles[] = $collaboratedBundle;
```

Comment:
This implementation is broken: you only update the inverse side of the relation, which means that Doctrine will not persist this change.

And does the relation actually need to be bidirectional ? Unidirectional relations are much easier to reason about and to handle.